README.md

```diff
@@ -23,23 +27,49 @@ lazy val app = (project in file("app"))
   .enablePlugins(AkkaSerializationHelperPlugin)
 ```
 
-## Features
+## Missing serialization binding
 
-### 1. Serializability Checker
+To serialize message, persistent state or event in Akka, Scala trait needs to be defined:
 
-A Scala compiler plugin that detects messages, events and persistent states, and checks whether they extend the given base
-trait and report an error when they don't. This ensures that the specified serializer is used by Akka and protects
-against an accidental use
-of [Java serialization](https://doc.akka.io/docs/akka/current/serialization.html#java-serialization) or outright serialization failure.
+```scala
+package org
+trait MySer
+```
 
-To use, annotate your base trait with `@org.virtuslab.ash.SerializabilityTrait`:
+Also, a serializer needs to be defined for Akka in Scala configuration file:
```

Comment:
```suggestion
Also, a serializer needs to be bound to this trait in a configuration file:
```

Reply:
(HOCON-formatted config files aren't specific to Scala after all, their API is Java-compatible)